plugins/dev-tools/src/field_test_helpers.mocha.js

```diff
@@ -9,45 +9,55 @@ import {runTestCases, TestCase} from './common_test_helpers.mocha';
 
 /**
  * Field value test case.
- * @extends {TestCase}
+ * @implements {TestCase}
  * @record
  */
-export function FieldValueTestCase() {}
-FieldValueTestCase.prototype = new TestCase();
-/**
- * @type {*} The value to use in test.
- */
-FieldValueTestCase.prototype.value = undefined;
-/**
- * @type {*} The expected value.
- */
-FieldValueTestCase.prototype.expectedValue = undefined;
-/**
- * @type {string|undefined} Optional expected text (if not specified, default is
- *    String(expectedValue).
- */
-FieldValueTestCase.prototype.expectedText = undefined;
-/**
- * @type {!RegExp|string|undefined} Optional error message matcher if test case
- *    is expected to throw.
- */
-FieldValueTestCase.prototype.errMsgMatcher = undefined;
+export class FieldValueTestCase {
+  /**
+   * Class for a a field value test case.
+   */
```

Comment:
No need to change this, but I learned the other day that [descriptions are optional for constructors](https://google.github.io/styleguide/jsguide.html#jsdoc-class-comments) (you still need to provide parameter annotations though).